src/Cpanel/PluginActions.php

```diff
@@ -21,4 +51,51 @@ public function applyDefaultSettings()
     {
         // Do nothing
     }
+
+    public function getConfig()
+    {
+        $this->getUserConfig();
+        $this->getComposerJson();
+
+        //Clone the config to manipulate
+        $config = array_merge(array(), self::$CONFIG);
```

Comment:
I'm not sure if `array_merge` does deep copy. You might need to use `array_merge_recursive`.